src/parser/generated/regexp-tree.js

```diff
@@ -754,10 +754,33 @@ function getRange(text) {
  * Creates a character node.
  */
 function Char(value, kind, loc) {
+  let symbol;
+
+  switch (kind) {
+    case 'decimal': {
+      const code = parseInt(value.slice(1));
```

Comment:
We can use `Number(...)` coercion, it's faster, and does exactly coercion instead of parsing. Or otherwise, we need to put explicit `10` as radix for `parseInt`.